src/common/translations/localize.ts

```diff
@@ -116,17 +94,33 @@ export const computeLocalize = async <Keys extends string = LocalizeKeys>(
       } catch (err: any) {
         return "Translation error: " + err.message;
       }
-      cache._localizationCache[messageKey] = translatedMessage;
+      localizationCache[translatedValue] = translatedMessage;
     }
 
     let argObject = {};
+    // TS says that we will always get the new format, but we might get the old format
+    const args = _args as any;
     if (args.length === 1 && typeof args[0] === "object") {
       argObject = args[0];
-    } else {
+    } else if (args.length >= 2) {
+      // eslint-disable-next-line no-console
+      console.warn(
+        "[FIXME] While localizing",
```

Comment:
Can we get rid of the "[FIXME]" please and stick to a readable message?

Reply:
Also I'd prettify and make it something like:

```suggestion
        "Old format for parameters was found while localizing key %s.  Please see ...", key);
```

Reply:
> and stick to a readable message

Could you clarify how this isn't readable?

Reply:
Sure... "FIXME" is not a word, all caps is more difficult to read even if it was, and it's not helpful to anyone seeing it, especially users.

What I wrote puts the key into a couple of clear sentences.

Reply:
A problem with your wording is that it isn't a problem with the key, it's a problem with where it was called.